ci-operator/step-registry/sandboxed-containers-operator/get-kata-rpm/sandboxed-containers-operator-get-kata-rpm-commands.sh

```diff
@@ -8,15 +8,18 @@
 
 cd /tmp || exit 1
 
-# Read from secrets
-KATA_RPM_BASE_URL=$(cat /usr/local/sandboxed-containers-operator-ci-secrets/secrets/KATA_RPM_BASE_URL)
-
 arch=$(uname -m)
-ver=$(echo "$KATA_RPM_VERSION" | cut -d- -f1)
-build=$(echo "$KATA_RPM_VERSION" | cut -d- -f2)
-brew_auth="$(oc get -n openshift-config secret/pull-secret -ojson  | jq -r '.data.".dockerconfigjson"' |  base64 -d | jq -r '.auths."registry.redhat.io".auth' | base64 -d)"
+if [ -n "${KATA_RPM_BUILD_TASK}" ];then
+    KATA_RPM_BASE_URL=$(cat /usr/local/sandboxed-containers-operator-ci-secrets/secrets/KATA_RPM_BUILD_TASK_BASE_URL)
```

Comment:
Unless we need this later I'd suggest using a different name for this variable in this and the non-rpm-build branch as it took me a while to realize one points to BUILD_TASK_BASE_URL and one to BASE_URL... (I was banging my head against the wall not seeing the numbered locations on BASE_URL)

Reply:
done